js/snake.js

```diff
@@ -1,305 +1,417 @@
-// DBI Farben: Blau: #1E345A, Rot:
-const DBIRED = "#EE2E31";
-const DBIBLUE = "#1E345A";
-const DBIWHITE = "#FFFFFF"; //TODO checken ob es wirklich FFFFFF ist
-const GRIDSIZE = 15;
-const BGCOLOR = DBIBLUE;
-const INITSPEED = 500;
-const SPEEDUP = 1.05;
-const SPEEDLIMIT = 50;
-
-function mod(num, div) {
-    return ((num%div)+div)%div;
+async function main() {
+    let game;
+    game = new SnakeGame(document.body, 3);
```

Comment:
Game würde ich persönlich nicht über zwei Zeilen aufziehen. Habe dir mal eine Suggestion dagelassen. :)

```suggestion
    let game = new SnakeGame(document.body, 3);
```